src/bakery.js

```diff
@@ -13,4 +19,13 @@ export default class Bakery {
     return value === '1';
   }
 
+  static getExpireDate(daysToHide) {
```

Comment:
Exactly as mentioned above already, let's not have another method we need to test and maintain. Esp. because it's also very opinionated on the unit.